Snowflake.Data.Tests/SFConnectionIT.cs

```diff
@@ -188,6 +188,7 @@ public void TestCrlCheckSwitchConnection()
         [Test]
         public void TestConnectString()
         {
+            var schemaName = "dlSchema_" + Guid.NewGuid().ToString().Replace("-", "_");

```

Comment:
Why not provide epoch time here instead? It's easier to sort, easier to find the last one/current one.

Reply:
Epoch time is not unique and can be skewed when the system is synchronising the time with some server. I know there is a very little chance that collision will happen, but I think using the UUID is a better approach here.

Reply:
To find the newest schemas you can use `describe database <db>;` and sort it by `created_on` column.